content/en/_index.html

```diff
@@ -40,6 +40,25 @@ <h3 class="section-head">
   </div>
 </div>
 
+<div id="overview" class="text-center">
+  <div class="container">
+    <h3 class="section-head">
+      <span class="border-bottom pb-2">Trusted by</span>
+    </h3>
+  <div class="container trusted-logo-container">
+    <img src="/docs/images/logos/Amazon_2024.svg" alt="Amazon" class="trusted-logo">
+    <img src="/docs/images/logos/ByteDance.svg" alt="ByteDance" class="trusted-logo">
```

Comment:
Please remove ByteDance as they are not a [CNCF member](https://landscape.cncf.io/?group=members), perhaps replace them with NVIDIA (pending approval from them and KSC):

```suggestion
```